lib/que.rb

```diff
@@ -3,14 +3,15 @@
 require 'socket' # For hostname
 
 module Que
-  autoload :Adapters,   'que/adapters/base'
-  autoload :Job,        'que/job'
-  autoload :Migrations, 'que/migrations'
-  autoload :SQL,        'que/sql'
-  autoload :Version,    'que/version'
-  autoload :Worker,     'que/worker'
-  autoload :Metrics,    'que/metrics'
-  autoload :Locker,     'que/locker'
+  autoload :Adapters,        'que/adapters/base'
+  autoload :Job,             'que/job'
+  autoload :Migrations,      'que/migrations'
+  autoload :SQL,             'que/sql'
+  autoload :Version,         'que/version'
+  autoload :Worker,          'que/worker'
+  autoload :Metrics,         'que/metrics'
+  autoload :Locker,          'que/locker'
+  autoload :JobTimeoutError, 'que/job_timeout_error'
```

Comment:
Not a fan of doing this given the strategy we have now of releasing a feature branch. Happy to revert updating the other lines and just appending this one, and having it stick out.

Reply:
This should be fine I think. I'm not a fan of autoloading any of this stuff really, but given we already have an autoload system I think this makes sense as a change.